src/target/parsers/cpu.cc

```diff
@@ -29,10 +29,12 @@ namespace parsers {
 namespace cpu {
 
 Optional<String> DetectSystemTriple() {
+  #ifdef TVM_INFO_USE_LLVM
   auto pf = tvm::runtime::Registry::Get("target.llvm_get_system_triple");
   if (pf->defined()) {
```

Comment:
You're correct, `pf->defined()` here doesn't really make sense because there will be a segfault when accessing the method. I'm happy to stick with the `#ifdef TVM_LLVM_VERSION` check, but we should probably add an `ICHECK` here as well, similar to other uses in the codebase e.g. https://github.com/apache/tvm/blob/bc1597ce2b643c20b0713ab546b7447459708505/src/relax/backend/contrib/cutlass/codegen.cc#L280